src/components/views/rooms/MessageComposer.tsx

```diff
@@ -15,6 +15,7 @@ limitations under the License.
 */
 
 import React, { createRef, ReactNode } from "react";
+import { richToPlain, plainToRich } from "@matrix-org/matrix-wysiwyg";
```

Comment:
Is this import safe to use here without the dependency on https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/FinalizationRegistry

Reply:
No you're right this is going to cause the same issue that was looked at this morning. I'm sorting out lint/TS errors then will sort this.